inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+# Inline Cache with GemFire
+
+Using an inline cache with a database can help improve performance, reduce database scaling limits, and provide greater flexibility in terms of data retrieval. 
+It is an important tool for any application that relies heavily on database access, and can help ensure that critical data is always available when it is needed.
+
+GemFire's inline cache is a mechanism for temporarily storing frequently accessed data in memory, rather than querying a database every time the data is needed. 
+This can significantly improve the performance of database-driven applications, especially when dealing with large amounts of data or complex queries.
+
+One of the main reasons to use GemFire with a database is to address scaling limits of the database itself. 
+As the size and complexity of a database grows, it can become increasingly difficult to maintain fast and efficient access to the data. 
+This is especially true in situations where multiple users or applications are simultaneously accessing the database, leading to contention and slowdowns.
+
+By using GemFire's inline cache, frequently accessed data can be stored in GemFire's memory, reducing the number of queries that need to be made 
+to the database. This can help alleviate some of the scaling limits of the database, as it reduces the amount of load placed on the database server. 
+It can also help reduce network latency and improve overall system performance, as data can be retrieved locally rather than over a network connection.
+
+# Implementation
+Setting up an inline cache with GemFire and a database involves several steps. Here's an overview of the process.
+
+## Install and Configure GemFire
+
+The first step is to install and configure GemFire on your system.
+Download and install VMware GemFire from [Tanzu Network](https://network.pivotal.io). Follow the installation instructions in the [GemFire documentation](https://docs.vmware.com/en/VMware-GemFire/index.html).
```

Comment:
This should be using https://network.tanzu.vmware.com/ not the network.pivotal.io.
Also, we should link directly to the GemFire installation instructions (https://docs.vmware.com/en/VMware-GemFire/10.0/gf/getting_started-installation-install_intro.html) instead of dropping users on the index page and forcing them to find the install instructions on their own.

Reply:
updated